.streamlit/config.toml

```diff
@@ -0,0 +1,5 @@
+[browser]
+gatherUsageStats = false
+
+[theme]
+base = "dark"
```

Comment:
It seems to me that this file must be in the directory with the application i.e. `docs/backends/app/.streaamlitt/config.toml`

Reply:
I read the docs as saying it must be in the directory the application is run from, which I assumed to be the repo:

![image](https://user-images.githubusercontent.com/417981/212389370-8977b2ab-bf9d-42e8-ad7f-95b736c959b4.png)